index.html

```diff
@@ -72,8 +72,198 @@ <h1 class="intro">
         <img src="./Assets/Ellipse4.png" class="roll_ball" alt="roll_ball" />
         <img src="./Assets/Rectangle5.png" class="roll_bar" alt="roll_bar" />
         <img src="./Assets/Ellipse12.png" class="bowl" alt="bowl" />
-        <img src="./Assets/Ellipse13.png" class="hook" alt="hook"/>
+        <img src="./Assets/Ellipse13.png" class="hook" alt="hook" />
       </section>
     </div>
+    <section class="works-container">
+      <div class="work-item">
+        <img
+          class="work-card"
+          src="./Assets/restaurant.JPG"
+          alt="project card display"
+        />
+        <div class="description-container">
+          <h1>Ristorant Confusion</h1>
+          <p class="description">
+            A Restaurant publicity and order website<br />
+            We take inpiration from the World's best Cuisine to create a
+            lipsmacking and satisfying collection
+          </p>
+          <ul class="pale-info-container">
+            <li class="pale-info">html</li>
+            <li class="pale-info">Css</li>
+            <li class="pale-info">JavaScript</li>
+          </ul>
+          <div class="action">
+            <button class="see-project" type="button">See Project</button>
```

Comment:
- [ ] [**OPTIONAL**] Let's make this button more interactive by adding the CSS `Pressed`, `Hover`, and `Active` button styling effects.